main.tex

```diff
@@ -709,13 +709,20 @@ \subsubsection{Tables}
 \label{sec:table}
 % fleshed out / edited by Adrian PW
 
-The \package{astropy.table} subpackage provides functionality for representing
-and manipulating heterogeneous data.
-The package was described in detail in \cite{astropy}.
-Thus, in what follows, we only summarize key new features or updates to
-\package{astropy.table} since the previous \astropy paper.
+The \package{astropy.table} subpackage provides functionality for
+representing and manipulating heterogeneous data. In some respects,
+this is similar to \package{numpy} record arrays \citep{numpy} or
+\package{pandas} dataframes \citep{pandas} but with modifications for
+astronomical data. Most notably, tables from \package{astropy.table}
+allow for table or column metadata and can handle
```

Comment:
Can we use capitized `Table` and `Column` here?

Reply:
We could, but I did not intend them to mean `Table` as in "instance of the table class", I actually mean: Metadata that is associated with a table (which then in astropy is implemented as a dictionary called "meta" that is an attribute of Table objects).

Reply:
Just to be clear, we could talk about `Table` and `Column` as objects in Python code, but I don't see what we gain from them here. (Saying: "they allow for `Table` and `QTable`, as well as `Column` and `MaskedColumn` metadata" ist just not as compact as saying" table and column", where tables and column used independent of the implementation details.

Reply:
So, I'll leave it as it is for now.

Reply:
oh, this sounds fine, thanks for clearing it up.